components/properties-panel/node-property-renderer.tsx

```
@@ -1169,6 +1169,10 @@ export function NodePropertyRenderer({
         </div>
       );
 
+    case "materialNode":
+      const { MaterialEditor } = require("./property-editors/material-editor");
+      return <MaterialEditor properties={properties} setProperties={setProperties} />;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)